src/data-workspace/data-details-sidebar/limits-display.js

```diff
@@ -76,13 +78,20 @@ export default function LimitsDisplay({
                     </div>
                 )}
             </div>
-
             <ButtonStrip>
-                <EditButton onClick={onEditClick} />
-                <LimitsDeleteButton
-                    dataElementId={dataElementId}
-                    categoryOptionComboId={categoryOptionComboId}
-                />
+                {canAdd && <EditButton onClick={onEditClick} />}
```

Comment:
the UpdateLimits component opened by going to edit mode has a delete button, but it seemed strange to let users with delete authorities open the edit component just in order to have access to the delete button there (they already have access to delete button in this component). Anyway, I've hidden EditButton if people don't have add authority.